src/Core/Stock/StockManager.php

```diff
@@ -188,10 +189,14 @@ public function updateQuantity($product, $id_product_attribute, $delta_quantity,
     public function saveMovement($productId, $productAttributeId, $deltaQuantity, $params = array())
     {
         if ($deltaQuantity != 0) {
-            $movement = $this->prepareMovement($productId, $productAttributeId, $deltaQuantity, $params);
+            $stockMvt = $this->prepareMovement($productId, $productAttributeId, $deltaQuantity, $params);
 
-            if ($movement) {
-                return $this->registerMovement($movement);
+            if ($stockMvt) {
+                global $kernel;
+                if (!is_null($kernel)) {
```

Comment:
I just want be sure to understand, if symfony is not instantiated, we cannot save?